# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/test/gtest/parent/TestFileSystemOriginInitialization.cpp
# Commit: b84880c5de44
# Full Hash: b84880c5de443e6274953d210444c57319ce4b5f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-13 09:18:36
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Make it possible to initialize temporary origins without ensuring origin directories; r=dom-storage-reviewers,jari
#   
#   LSNG already uses some QuotaManager APIs to achieve that origin directories are
#   not created if they don't exist during datastore preparation, but the feature
#   is not easy to use and it's also not generalized enough for use in other quota
# ==============================================================================

diff -r c5f28ed5382b -r b84880c5de44 dom/fs/test/gtest/parent/TestFileSystemOriginInitialization.cpp
--- a/dom/fs/test/gtest/parent/TestFileSystemOriginInitialization.cpp	Sat Oct 12 23:42:06 2024 +0300
+++ b/dom/fs/test/gtest/parent/TestFileSystemOriginInitialization.cpp	Sat Oct 12 20:58:36 2024 +0000
@@ -131,7 +131,8 @@
   // Initialize origin
   ASSERT_NO_FATAL_FAILURE(InitializeStorage());
   ASSERT_NO_FATAL_FAILURE(InitializeTemporaryStorage());
-  ASSERT_NO_FATAL_FAILURE(InitializeTemporaryOrigin());
+  ASSERT_NO_FATAL_FAILURE(
+      InitializeTemporaryOrigin(/* aCreateIfNonExistent */ true));
 
   // After initialization,
   // * origin usage is nothing